CHANGES/8594.bugfix

```diff
@@ -0,0 +1 @@
+Handle skipping of corrupted docker content that has futures.
```

Comment:
How about `Handled properly skipping of corrupted docker content.`?
Past tense for changelog entries + I think users have no idea what futures are, so I'm trying to make the changelog entry more user oriented.